src/components/biggive-cookie-banner/biggive-cookie-banner.tsx

```diff
@@ -23,17 +23,16 @@ export class BiggiveCookieBanner {
   /**
    * Indicates that the user has made a selection of cookies purpose to accept.
    *
-   * Event data contains an array of the type of cookie purposes the user consents to. Currently,
-   * it is only possible to consent to marketing cookies, so it will simply be ['marketing'] or [] but
-   * in future other types may be possible.
+   * Event data contains an array of the type of cookie purposes the user consents to, e.g.
+   * [] or ['a-and-t', 'third-party']
```

Comment:
just realised this describes a structure that's completely different to what the code is actually doing and what the types say. The array thing must be an idea I had a couple of weeks ago that got left in the comment but not kept in the code.